meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/redsocks.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
If you want to do that, I would suggest to write a service in the redsocks recipe and add a drop in the this recipe which modifies something (deps probably).